platform/upgrades/adding-custom-cloud-credentials.md

```diff
@@ -16,6 +16,9 @@ REQUEST access to this feature! message on community slack: gridai-community.sla
 Using your own AWS credentials with Grid is currently not available in the Web UI; see below for CLI instructions.
 {% endhint %}
 
+An Amazon EC2 [Auto Scaling group (ASG)]("https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html") contains a collection of machines that share similar characteristics and are treated as a logical grouping for the purposes of fleet management and dynamic scaling.
```

Comment:
They'd also need to increase their:
* Lunch template quotas
* Instance profile quotas
* Lunch configuration quotas
* EC2 instance quotas

Reply:
Also let them increase to a 1000 just to be safe